odoo/addons/test_translation_import/tests/test_term_count.py

```diff
@@ -20,8 +20,29 @@ def test_count_term(self):
         ids = self.env['ir.translation'].search([
```

Comment:
Maybe it is time to rename the old-API-ish `ids` to `trans` or a more meaningful name?